Vegeting/Screens/PostDetail/ProfileCollectionViewCell.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ProfileCollectionViewCell.swift
+//  Vegeting
+//
+//  Created by 김원희 on 2022/10/26.
+//
+
+import UIKit
+
+private enum Constants {
+    static let profileImageSize = 80.0
+    static let profileImageCGSize = CGSize(width: Constants.profileImageSize, height: Constants.profileImageSize)
+    static let borderWidth = 2.0
+    static let spacing = 4.0
+}
+
+struct ParticipantsInfo {
+    let profileImage: UIImage!
+    let participantsName: String
+}
+
+class ProfileCollectionViewCell: UICollectionViewCell {
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        configureAddSubviews()
+        setupLayout()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    let profileImage: UIImageView = {
+        let image = UIImageView()
+        image.layer.cornerRadius = (Constants.profileImageSize - Constants.spacing * 2) / 2.0
+        image.clipsToBounds = true
+        return image
+    }()
+    
+    let participantsName: UILabel = {
+        let label = UILabel()
+        label.font = .preferredFont(forTextStyle: .subheadline)
+        label.textAlignment = .center
+        return label
+    }()
```

Comment:
p1;
해당 부분은 private으로 바꿔주세요.
```suggestion
    private let profileImage: UIImageView = {
        let image = UIImageView()
        image.layer.cornerRadius = (Constants.profileImageSize - Constants.spacing * 2) / 2.0
        image.clipsToBounds = true
        return image
    }()
    
    private let participantsName: UILabel = {
        let label = UILabel()
        label.font = .preferredFont(forTextStyle: .subheadline)
        label.textAlignment = .center
        return label
    }()
```

Reply:
[수정](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Vegeting/pull/44/files#diff-7bbaa97d7a207de1f1a86e75d64a314f7a35d3e349fd067442baad4468e81ef1R35-R47)하였습니다~